rpcs3/Emu/RSX/VK/VKHelpers.h

```diff
@@ -2511,8 +2511,15 @@ namespace vk
 			instance_info.ppEnabledExtensionNames = fast ? nullptr : extensions.data();
 
 			VkInstance instance;
-			if (vkCreateInstance(&instance_info, nullptr, &instance) != VK_SUCCESS)
+			VkResult result = vkCreateInstance(&instance_info, nullptr, &instance);
+			if (result == VK_ERROR_LAYER_NOT_PRESENT)
```

Comment:
You can put this check in the scope of the other

Reply:
Agreed. Avoids checking result twice when everything is fine.